plasmapy/diagnostics/thomson.py

```diff
@@ -936,6 +970,9 @@ def spectral_density_model(wavelengths, settings, params):
             "`numpy.delete`."
         )
 
```

Comment:
Add to docstring of `spectral_density_model`

Reply:
The notches keyword still needs to be added to the list of `settings` in the docstring for the model function.